libs/components/core/src/lib/modules/overlay/overlay.component.scss

```diff
@@ -15,6 +15,8 @@
 
     &-fixed {
       position: fixed;
+      overflow: hidden;
+      height: calc(var(--skyux-vh, 1vh) * 100);
```

Comment:
This is 2/2 of the fix for mobile Safari, which was the original issue where we tried using absolute position.